### PR TITLE
Update amo-validator dependencies

### PR DESCRIPTION
**Pull request by wagnerand**
This depends on https://github.com/mozilla/amo-validator/pull/455

### CODE REVIEW

File: requirements/prod.txt [hunk 1]
```diff
@@ -184,24 +191,37 @@ kombu==3.0.35 \
...
+    --hash=sha256:62648ac68cf55193bcb4c1078cf99dfb72bb7ed6c1bacca5fd36b6167952b5c6
 ndg-httpsclient==0.4.1 \
     --hash=sha256:133931ab2cf7118f8fc7ccc29e289ba8f644dd90f84632fa0e6eb16df4ba1891
 newrelic==2.66.0.49 \
     --hash=sha256:f95b90def0c86b6d4f859664ca411bae90da76345f23ed2de40531d39e2b32a1
 nobot==0.4.1 \
     --hash=sha256:e835abfe9b813fe5697475e8fc0dfbaf9c7572f4574e3d2d7e5114bc84f4cf0b \
     --hash=sha256:bae7bc785e81d4edcc7e7e66457697995c47475e0a2aba405b3a5cb82697e880
+# nose is required by amo-validator
+nose==1.3.7 \
```
Comment (diox) [inline requirements/prod.txt]: We should convert amo-validator to using py.test to be able to remove this one. It's probably straightforward.

**Comment by diox** [comment]
r+wc

**Comment by wagnerand** [comment]
Ok, I will do do that in a future PR.

**Comment by wagnerand** [comment]
Thank you!

**Pull request merged**